Sources/QuickTests/FunctionalTests/PendingTests.swift

```diff
@@ -45,13 +45,13 @@ class PendingTests: XCTestCase, XCTestCaseProvider {
         oneExampleBeforeEachExecutedCount = 0
 
         qck_runSpec(FunctionalTests_PendingSpec.self)
-        XCTAssertEqual(oneExampleBeforeEachExecutedCount, 1)
+        XCTAssertEqual(oneExampleBeforeEachExecutedCount, 2)
     }
 
     func testBeforeEachDoesNotRunForContextsWithOnlyPendingExamples() {
         onlyPendingExamplesBeforeEachExecutedCount = 0
 
         qck_runSpec(FunctionalTests_PendingSpec.self)
-        XCTAssertEqual(onlyPendingExamplesBeforeEachExecutedCount, 0)
+        XCTAssertEqual(onlyPendingExamplesBeforeEachExecutedCount, 1)
```

Comment:
This assertion no longer matches the name of the test. The test is named "test that `beforeEach` does not run for `context`s with only `pending` examples", whereas this assertion verifies that `beforeEach` **does** run. The same goes for `testBeforeEachOnlyRunForEnabledExamples` above.

So, why do these tests exist? `pending` examples were first implemented in #38. At that time, Quick had an ad hoc test suite: a single test file that used Quick and was expected to pass. I added in-depth tests in #154, which included tests for `pending` in 6b7681b6d78e8c4944fe91ad81b49267aaf59821. Unfortunately, I neglected to realize the importance of documenting in the commit message whether `beforeEach` not running was intended, or merely the behavior at the time that I wrote these tests.

I think these tests merely documented the behavior of `pending` at the time. It seems like the purpose of this PR is to ensure that `beforeEach`, `afterEach`, `beforeSuite`, and `afterSuite` always run, even for `pending` examples. If that's the case, please:

1. Delete the `testBeforeEachOnlyRunForEnabledExamples` and `testBeforeEachDoesNotRunForContextsWithOnlyPendingExamples` tests.
2. Add tests for `beforeEach`, `afterEach`, `beforeSuite`, and `afterSuite`.